protocol/x/clob/abci.go

```diff
@@ -156,11 +164,26 @@ func PrepareCheckState(
 	)
 
 	// 3. Place all stateful order placements included in the last block on the memclob.
+	// Note telemetry is measured outside of the function call because `PlaceStatefulOrdersFromLastBlock`
+	// is called within `PlaceConditionalOrdersTriggeredInLastBlock`.
+	startPlaceLongTermOrders := time.Now()
 	offchainUpdates = keeper.PlaceStatefulOrdersFromLastBlock(
 		ctx,
 		processProposerMatchesEvents.PlacedLongTermOrderIds,
 		offchainUpdates,
 	)
+	telemetry.MeasureSince(
+		startPlaceLongTermOrders,
+		types.ModuleName,
+		metrics.PlaceLongTermOrdersFromLastBlock,
+		metrics.Latency,
+	)
+	telemetry.SetGauge(
+		float32(len(processProposerMatchesEvents.PlacedLongTermOrderIds)),
+		types.ModuleName,
+		metrics.PlaceLongTermOrdersFromLastBlock,
+		metrics.Count,
+	)
```

Comment:
nit: I personally would put these inside `PlaceStatefulOrdersFromLastBlock`

Reply:
Personally I avoided it since I wanted to differentiate between long-term order placements and stateful order placements. Let me know if you disagree though and I can change it.